.vscode/settings.json

```diff
@@ -0,0 +1,11 @@
+{
+  "files.exclude": {
+    "amplify/.config": true,
+    "amplify/**/*-parameters.json": true,
+    "amplify/**/amplify.state": true,
+    "amplify/**/transform.conf.json": true,
+    "amplify/#current-cloud-backend": true,
+    "amplify/backend/amplify-meta.json": true,
+    "amplify/backend/awscloudformation": true
+  }
+}
```

Comment:
This should probably ignored, see the `.gitignore` doesn't exclude the whole folder. Could you maybe just update it to ignore the whole `.vscode` folder?